src/components/new-request/search.vue

```diff
@@ -40,17 +40,16 @@
         <EntityType />
         <CompanyType v-if="getEntityTypeCd && isNumberedEntityType" />
 
-        <!-- named company -->
-        <template v-if="isNamedCompany || isSociety">
+        <template v-if="(isNamedCompany || !isNumberedEntityType) && entity_type_cd">
           <v-col cols="12" :md="showDesignation ? '8' : '12'">
-            <NameInput @emit-corp-num-validity="corpNumValid = $event" />
+            <NameInput
+              :is-mras-search="(isXproFlow && isMrasJurisdiction && !getHasNoCorpNum)"
+              @emit-corp-num-validity="corpNumValid = $event"
+            />
           </v-col>
-
           <Designation v-if="showDesignation" cols="12" md="4" />
         </template>
-
-        <!-- numbered company -->
-        <NumberedCompanyBullets v-if="isNumberedCompany" />
+        <NumberedCompanyBullets v-if="isNumberedCompany && isNumberedEntityType" />
```

Comment:
Bullets are not displayed correctly with only one condition. Specific case is change Business type from Limited Numbered to COOP

Selected Limited Numbered:
![image](https://github.com/bcgov/namerequest/assets/116035339/aa21aca3-f288-4e1d-9e46-64559a96b2ce)

Changed to COOP:
![image](https://github.com/bcgov/namerequest/assets/116035339/e97f0c59-f86f-4458-81ca-2f434cc0af52)

Reply:
This is the same thing I fixed in this commit:
https://github.com/bcgov/namerequest/pull/691/commits/f3aa9799f088da3a73ca21c6b7403f4b55367423

Reply:
ohhh okay. I will take a look

Reply:
The conditional to display correct flow can be one of the below.

`<NumberedCompanyBullets v-if="!isNamedCompany && isNumberedEntityType" />`
`<NumberedCompanyBullets v-if="isNumberedCompany && isNumberedEntityType" />`

With keeping only one condition (isNumberedCompany  or isNumberedEntityType) different test scenario fail.

Please let me know if missing something or if there is a better way to handle this. :)

Reply:
Your first option is the negative of the conditional I implemented in my commit, above (which makes sense since I had it on the named block instead of the numbered block).

Your first option is what you already have.

How about simplifying everything, like this?
```
<template v-if="getEntityTypeCd">
  <!-- named company -->
  <template v-if="(isNamedCompany || !isNumberedEntityType)">
    <v-col cols="12" :md="showDesignation ? '8' : '12'">
      <NameInput
        :is-mras-search="(isXproFlow && isMrasJurisdiction && !getHasNoCorpNum)"
        @emit-corp-num-validity="corpNumValid = $event"
      />
    </v-col>
  </template>

  <!-- numbered company -->
  <NumberedCompanyBullets v-else />
</template>
```

Reply:
Okay. I get it what you meant